Google.PowerShell.IntegrationTests/Storage/Storage.GcsObject.Tests.ps1

```diff
@@ -640,8 +640,35 @@ Describe "Write-GcsObject" {
         Read-GcsObject $bucket $objectName | Should BeExactly $objectContents
 
         # Exercise the explicit -Content parameter too.
-        Write-GcsObject $bucket ($objectName + "2") -Contents $objectContents -Force
+        Write-GcsObject $bucket ($objectName + "2") -Value $objectContents -Force
         Read-GcsObject $bucket ($objectName + "2") | Should BeExactly $objectContents
+
+        # Using piped in GCS Object with -Value.
```

Comment:
This is no longer testing "accept contents form the pipeline". Make it its own `It` statement.